acl_setup.php

```diff
@@ -174,6 +181,25 @@
  $gacl->add_object('patients', 'Lab Results (write,addonly optional)','lab', 10, 0, 'ACO');
      // xl('Lab Results (write,addonly optional)')
 
+
+ $gacl->add_object('groups', 'Add/Update groups','gadd', 10, 0, 'ACO');
+ // xl('Add update groups (view ,write)')
+
+ $gacl->add_object('groups', 'Create/Update groups appointment in calender','gcalendar', 10, 0, 'ACO');
+// xl('Using calendar add edit event with groups (view ,write)')
```

Comment:
Regarding this and the next 3 xl() instances: I believe the intent is to exactly duplicate the ACO title so that it will be recorded as a translation constant by the script that examines code and picks out xl() and related calls. So in this example you need:
    // xl('Create/Update groups appointment in calender')